src/_guides/language/effective-dart/usage.md

```diff
@@ -119,6 +119,60 @@ library. Follow these two rules:
 * An import path should never contain `/lib/`.
 * A library under `lib` should never use `../` to escape the `lib` directory.
 
+## Booleans
+
+### DO use `??` to convert `null` to a Boolean value.
+
+You sometimes have an expression that can evaluate to `true`, `false`, or `null`
```

Comment:
This sounds presumptuous. How about:

This rule applies when an expression can evaluate ..., and you need to pass ....